modular_skyrat/code/modules/vending/kinkmate.dm

```diff
@@ -55,4 +54,4 @@
 /obj/item/vending_refill/kink
 	machine_name 	= "KinkMate"
 	icon_state 		= "refill_kink"
-*/
+
```

Comment:
Don't do it like this. Remove everything but the path of the kinkmate machine and product list.